install-wings.sh

```diff
@@ -586,10 +535,10 @@ function main {
     if [[ "$CONFIRM_FIREWALL_CMD" =~ [Yy] ]]; then
       CONFIGURE_FIREWALL_CMD=true
     fi
-
-    ask_letsencrypt
   fi
 
+  ask_letsencrypt
```

Comment:
This means all supported distros will at this stage try to install Let's Encrypt if the user opts in to do so. Have you verified that all supported distros and distro releases work with the Let's Encrypt the way we intended it?

Reply:
As far as I know Debian 9,10; Ubuntu 18.04, 20.04; CentOS 7, 8 works.

I think that's all the supported OS's.

Reply:
Have you tried the Let's Encrypt function on the supported distros? I am not questioning the fact that you are probably right, when we supported Ubuntu 16 this was not the case, hence the version compatibility checking was implemented in the first place.

Reply:
When I did the testing I did it on all OS's above. That's why I said they all work. When I test I also test ssl just in case I did something wrong there.

Reply:
Alright, all cool!

Reply:
Cool. cool.